pathway-analysis/02-gene_set_enrichment_analysis.Rmd

```diff
@@ -67,18 +63,37 @@ if (!dir.exists(output_dir)) {
 ```{r input_files}
 # DGE results
 dge_results_file <- file.path(input_dir,
-                             "nb_cell_line_mycn_amplified_v_nonamplified.tsv")
+                             "NB-cell_DESeq_amplified_v_nonamplified_results.tsv")
 ```
 
 #### Output files
 
 ```{r output_files}
 # GSEA pathway-level scores and statistics
 gsea_results_file <- file.path(output_dir,
-                               "nb_cell_line_gsea_results.tsv")
+                               "NB-cell_gsea_results.tsv")
 ```
 
-## Gene sets
+## Gene Set Enrichment Analysis
+
+_Adapted from [refine.bio examples](https://github.com/AlexsLemonade/refinebio-examples/blob/33cdeff66d57f9fe8ee4fcb5156aea4ac2dce07f/03-rnaseq/pathway-analysis_rnaseq_02_gsea.Rmd)_
+
+![](diagrams/subramanian_fig1.jpg)
+
+**Figure 1. [Subramanian _et al._ (2005)](https://doi.org/10.1073/pnas.0506580102).**
+
+GSEA calculates a pathway-level metric, called an enrichment score (sometimes abbreviated as ES), by ranking genes by a gene-level statistic.
+This score reflects whether or not a gene set or pathway is overrepresented at the top or bottom of the gene rankings ([Subramanian _et al._ 2005](https://doi.org/10.1073/pnas.0506580102); [Yu](http://yulab-smu.top/clusterProfiler-book/chapter2.html#gene-set-enrichment-analysis)) 
+
+Specifically, genes are ranked from most positive to most negative based on their statistic and a running sum is calculated by starting with the most highly ranked genes and increasing the score when a gene is in the pathway and decreasing the score when a gene is not.
+In this example, the enrichment score for a pathway is the running sum's maximum deviation from zero.
```

Comment:
Here I am trying to make this a bit more clear, but I might be failing. 

One thing I am doing is not calling anything a "score" but the maximum deviation, because I think that was something that threw me off. I also just aesthetically didn't like the double "and".

Also, I wasn't sure about the phrase "In this example"; I deleted it because it seems like this would be true always (assuming you accept that the maximum deviation can be negative)? Or is there some aspect here I am still misunderstanding?

```suggestion
Specifically, all genes are ranked from most positive to most negative based on their statistic and a running sum is calculated:
Starting with the most highly ranked genes, the running sum increases for each gene in the pathway and decreases for each gene not in the pathway.
The enrichment score for a pathway is the running sum's maximum deviation from zero.
```